src/components/containers/ConnectedMintBurnForm/ConnectedMintBurnForm.tsx

```diff
@@ -108,8 +109,9 @@ const ConnectedMintBurnForm: React.FunctionComponent<ConnectedMintBurnFormProps>
         transactionId={transactionId} 
         onComplete={handleComplete}
         notional={form.state.notional}
-        margin={Math.abs(form.state.margin as number) * (form.isRemovingMargin ? -1 : 1) }
-        onBack={handleGoBack} 
+        margin={isUndefined(form.state.margin && form.isRemovingMargin) ? 0 : 
```

Comment:
hmm, is `isUndefined(form.state.margin && form.isRemovingMargin)` intended (both in `isUndefined`)?